docs/docs/glossary/npm.md

```diff
@@ -0,0 +1,57 @@
+---
+title: npm or Node Package Manager
+disableTableOfContents: true
+---
+
+Learn what _npm_ is, how to use it, and how it fits in to the Gatsby ecosystem.
+
+## What is npm?
+
+<abbr>
+  npm
+</abbr>, or Node Package Manager, is the default package manager for the [Node.js](/docs/glossary/node) JavaScript runtime. It lets you install and update libraries and frameworks (dependencies) for Node-based projects, and interact with the npm Registry. You'll use npm to install and upgrade Gatsby and its plugins.
+
+npm is a [command line](/docs/glossary#command-line) tool. You'll need Terminal (Mac, Linux) or Command Prompt (Windows) in order to run its commands. To use one of npm's features, type `npm <command>` . For example, `npm help` displays a list of available features, including `install`, `uninstall`, `update`, and `search`.
+
+npm is installed alongside Node during the default [installation process](/tutorial/part-zero/#install-nodejs-for-your-appropriate-operating-system). You don't need to take any additional steps to add it to your environment.
+
+### Using npm to install Gatsby
+
+To install Gatsby, use the `npm install` command. Since Gatsby needs to be installed globally as a CLI app, you'll need to use the `--global` or `-g` flags.
+
+```shell
+npm install -g gatsby-cli
+```
+
+This command will install the Gatsby command line interface, or <abbr>CLI</abbr>. Once the installation completes, you can run `gatsby new my-project` to create a new Gatsby project.
+
+### Using npx to install Gatsby
```

Comment:
I started to write a comment about not _needing_ to install globally before seeing this section. The previous section could potentially use a little clarification on that point. Once a reader gets to this section, though, I think things become clearer.

Reply:
So instead of:

> Since Gatsby needs to be installed globally as a CLI app ...

Maybe this should be:

> It's recommended to install Gatsby as a CLI app. Doing so allows you to use the `gatsby` command to easily create new and manage new projects.

Reply:
@webinista I've added another suggestion which feels a bit more active/clear to me. Let me know what you think?

Reply:
Yeah. That's a better way to phrase it. Thanks!